src/main.cpp

```diff
@@ -721,8 +723,8 @@ void setup()
   {
     LOG(D, "Issuer ID: %s, Public Key: %s", utils::bufToHexString(issuer.issuerId, sizeof(issuer.issuerId)).c_str(), utils::bufToHexString(issuer.publicKey, sizeof(issuer.publicKey)).c_str());
   }
-  homeSpan.enableOTA();
-  homeSpan.begin(Category::Locks, "HK Lock");
+  homeSpan.enableOTA(OTA_PWD);
+  homeSpan.begin(Category::Locks, NAME);
```

Comment:
this will only set the MDNS display name, the name that would be shown once configured is set on `new Characteristic::Name("NFC Lock")`